src/main/scala/org/broadinstitute/dsde/workbench/sam/util/AsyncLogging.scala

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.dsde.workbench.sam.util
+
+import cats.effect.IO
+import com.typesafe.scalalogging.LazyLogging
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object AsyncLogging extends LazyLogging {
```

Comment:
I know logging is all side effects, but can we find a way to have some tests for this?

Reply:
That's really hard to do. There's no real mechanism for monitoring the stdout or stderr without a lot of setup. Given that these methods just call the logger, and that we can rely on Scala's type system to make sure they're not affecting whatever stream they're a part of, I'm ok with not specifically testing them.